src/pages/gathering-input.js

```diff
@@ -45,17 +46,73 @@ const Pages = () => (
       </div>
       <div className="Pages-exampleItem">
         <MultilineTextControl
+          errored
           hint="So that we can post the certificate"
           id="multiline-text-control-example"
           label="Address"
           size="48"
+          validation="Enter address"
           verticalSize="8"
-          errored
-          validation="Enter address as it is required"
+        />
+      </div>
+    </div>
+    <div className="Pages-headingOne">
+      <HeadingOne>{`SelectionControl`}</HeadingOne>
+    </div>
+    <div className="Pages-paragraph">
+      <Paragraph>{`If you're using SelectionControl:`}</Paragraph>
+    </div>
+    <div className="Pages-unorderedList">
+      <UnorderedList>
+        <ListItem>{`use instead of drop downs`}</ListItem>
```

Comment:
I'm not sure about this line. 

"If you're using SelectionControl use instead of drop downs". It doesn't seem to fit the rest of the advice.

Can that advice be hoisted up, I feel the reader should know that before reading the bit that assumes they've selected SelectionControl.

Reply:
Good spot. The next issue on our radar is about capturing multiple pieces of data. We intend to use that issue to revisit how we talk about components e.g. hoisting this kind of information up. This PR introduces the new component in the existing format.